api/resmap/resmap.go

```diff
@@ -532,11 +532,7 @@ func (m *resWrangler) ErrorIfNotEqualSets(other ResMap) error {
 	for _, r1 := range m.rList {
 		id := r1.CurId()
 		others := m2.GetMatchingResourcesByCurrentId(id.Equals)
-		if len(others) < 0 {
-			return fmt.Errorf(
-				"id in self missing from other; id: %s", id)
-		}
-		if len(others) > 1 {
+		if len(others) != 1 {
```

Comment:
How about changing `len(others) < 0 ` to `len(others) == 0`?

Reply:
> How about changing `len(others) < 0 ` to `len(others) == 0`?

Thanks. Your suggestion will make the error messages more detailed.
I'll fix this.